cascades-server/configuration/src/test/java/pl/gov/coi/cascades/server/presentation/PresentationConfigurationTest.java

```diff
@@ -66,23 +63,27 @@ public void testProduceDatabaseNameGeneratorService() throws Exception {
     }
 
     @Test
-    public void testProduceCredentials() throws Exception {
-        // given
-        PresentationConfiguration presentationConfiguration = new PresentationConfiguration();
-
+    public void testProduceCredentials() {
         // when
-        UsernameAndPasswordCredentialsGeneratorService actual = presentationConfiguration.produceCredentials();
+        UsernameAndPasswordCredentialsGeneratorService actual = presentationConfiguration.produceCredentials(new SecureRandom());
```

Comment:
Do not use `SecureRandom` in tests as it gives you no advantage and makes your tests run slower. Can be even, dramatically slower if you run out of entropy.